### PR TITLE
Added note for using GNOME tweaks to restore theme settings

### PR DESCRIPTION
**Pull request by rowansmithau**
Great that there is a page detailing how to use another DE, however upon removal of the new DE and switching back to Pop_!OS default shell various visual elements need to be fixed manually with GNOME tweaks, which I found out the hard way. This adds a note mentioning these steps in the hope it helps others.

### CODE REVIEW

**Review by Ampersandstorm — COMMENTED** [review]
Looks good, I've tweaked the wording and placed this contribution under the new `Reverting Back to Pop-desktop` header.

**Review by rowansmithau — COMMENTED** [review]
LGTM

File: content/desktop-environment.md [hunk 1]
```diff
@@ -37,14 +45,16 @@ For Ubuntu, install the `ubuntu-desktop` package instead:
...
+In some instances, icons and display settings may not automatically revert to their original values. To resolve this, install the GNOME Tweaks utility, navigate to the `Appearance` tab, then restore `Themes` settings to the `Pop` options. GNOME Tweaks can be installed from the Pop!\_Shop, or with this command:
 
-![Login2](/images/desktop-environment/Login2.png)
+```bash
+sudo apt install gnome-tweaks
+```
 
 ## Different Desktop Environments
 
+The desktop environment acts as the top graphical layer of the OS. The desktop environment is launched by a display manager; Pop!\_OS and Ubuntu both use GDM (GNOME Display Manager) by default.
```
Comment (storyaddict) [inline content/desktop-environment.md:56]: I suggest we merge this PR and then move this section further up on the article.

**Review by storyaddict — APPROVED** [review]
Good clarification to add.

**Pull request closed without merging**